src/test/resources/inputs/builtins/datetime.sql

```diff
@@ -2,13 +2,13 @@
 CURRENT_DATE;
 
 --#[datetime-01]
-DATE_ADD(SECOND, 5, CURRENT_DATE);
```

Comment:
self-review: previous tests using `CURRENT_DATE` (date value) as argument to `DATE_ADD` with a time field were wrong. changed to use a `TIME` value